libs/map/bookmark_helpers.cpp

```diff
@@ -353,6 +354,15 @@ std::string GenerateValidAndUniqueFilePathForGPX(std::string const & fileName)
   return GenerateUniqueFileName(GetBookmarksDirectory(), std::move(filePath), kGpxExtension);
 }
 
+std::string GenerateValidAndUniqueFilePathForGeoJson(std::string const & fileName)
+{
+  std::string filePath = RemoveInvalidSymbols(fileName);
+  if (filePath.empty())
+    filePath = kDefaultBookmarksFileName;
+
+  return GenerateUniqueFileName(GetBookmarksDirectory(), std::move(filePath), kGeoJsonExtension);
+}
```

Comment:
We have multiple functions `GenerateValidAndUniqueFilePathForKML`, `GenerateValidAndUniqueFilePathForGPX`, `GenerateValidAndUniqueFilePathForGeoJson`. Refactor is needed?